samples/dfu/nrf5340/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(lte_uart_update_53)
```

Comment:
Should the samples folder have a bit more descriptive name? `dfu_lte_uart` instead of just `dfu`? Only having `dfu` makes it sound like all DFU samples are located here, which they are not.

Reply:
IMO this needs to be addressed.